src/composables/auth.ts

```diff
@@ -1,47 +1,45 @@
 import { BiometricAuth } from '@aparajita/capacitor-biometric-auth';
 import {
   computed,
+  reactive,
   readonly,
   ref,
   watch,
 } from 'vue';
-import { generateMnemonic, mnemonicToSeed } from '@aeternity/bip39';
+import { generateMnemonic, mnemonicToSeed, validateMnemonic } from '@aeternity/bip39';
 
 import { tg as t } from '@/popup/plugins/i18n';
 import {
   IS_EXTENSION,
   IS_IOS,
   IS_MOBILE_APP,
   IS_OFFSCREEN_TAB,
-  MODAL_BIOMETRIC_LOGIN,
+  IS_WEB,
   MODAL_ENABLE_BIOMETRIC_LOGIN,
-  MODAL_PASSWORD_LOGIN,
   STORAGE_KEYS,
   UNFINISHED_FEATURES,
 } from '@/constants';
 import { STUB_ACCOUNT } from '@/constants/stubs';
 import {
-  authenticateWithPassword,
+  decodeBase64,
+  decrypt,
+  encodeBase64,
   encrypt,
-  endSession,
   generateEncryptionKey,
+  generateSalt,
   getSessionEncryptionKey,
-  sleep,
-  startSession,
+  sessionEnd,
+  sessionStart,
   watchUntilTruthy,
 } from '@/utils';
 
-import { WalletStorage } from '@/lib/WalletStorage';
-import { SecureMobileStorage } from '@/lib/SecureMobileStorage';
-
 import migrateMnemonicVuexToComposable from '@/migrations/002-mnemonic-vuex-to-composable';
 import migrateMnemonicCordovaToIonic from '@/migrations/008-mnemonic-cordova-to-ionic';
 
 import { useUi } from './ui';
 import { useModals } from './modals';
 import { createCustomScopedComposable } from './composablesHelpers';
-import { useSecureStorageRef } from './secureStorageRef';
```

Comment:
I think this composable should be removed within this PR.

Reply:
I will replace it with the new logic within separate PR I mentioned on the chat, OK?